spring-cloud-stream-binder-kinesis-docs/src/main/asciidoc/overview.adoc

```diff
@@ -138,14 +138,18 @@ createRetries::
     The amount of times the consumer will poll DynamoDB while waiting for the checkpoint table to be created
 +
 Default: `25`
+billingMode::
+    The Billing Mode of the DynamoDB table. See https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.ReadWriteCapacityMode.html#HowItWorks.OnDemand[DynamoDB On-Demand Mode]. Possible values are `provisioned` and `payPerRequest`. If left empty or set to `payPerRequest` both `readCapacity` and `writeCapacity` are ignored
```

Comment:
You need to revise docs since now we use `BillingMode` enum.
Although I won't mind if you introduce a local one to really support constants in lower case like we have for many other options in this project.

Reply:
I did try using these values and they worked as expected. I think the config parser is doing it already and no additional enum is required.

Reply:
Nice! Thanks for confirming.
Merging then!..